<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(118, 125, 129);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 127);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Выбор валюты</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>241</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(212, 212, 212);
font: 10pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 0, 0);
border-color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>Отобразить информацию </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_USD">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>доллар </string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_EURO">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Евро</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_RUB">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 170, 255);
font: 75 10pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Российский рубль</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>401</width>
      <height>691</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(85, 255, 127);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Exit">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>740</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 0);
background-color: rgb(204, 204, 204);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Вернуться назад</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Help">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>75</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 255, 255);
font: 11pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Справка</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>351</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
